api/db-cdr/generate-cdr/libproject/genomic_manifests.rb

```diff
@@ -30,28 +30,24 @@
 #
 # Intermediates:
 #
-# We generate one or more manifests of source/desintation GCS file paths, which
+# We generate one or more manifests of source/destination GCS file paths, which
 # describe how data will be copied.
 #
-# TODO(RW-8266): details on the publishing/copy process, details on manifest provenance
-#
 # See the CDR playbook for more context on the overall publishing process:
 # https://docs.google.com/document/d/1St6pG_EUFB9oRQUQaOSO7a9UPxPkQ5n4qAVyKF9j9tk/edit#heading=h.xt7avgt1nsoh
 
 PROD_BROAD_BUCKET = "gs://prod-drc-broad"
 OLD_ARRAYS_PATH_INFIX = "array_old_egt_files"
 
-
 CURATION_SYNTHETIC_SOURCE_CONFIG = {
-  :wgs_aw4_prefix => "gs://all-of-us-workbench-test-genomics/aw4_wgs/test_aw4.csv"
+  :wgs_aw4_prefix => "gs://all-of-us-workbench-test-genomics/aw4_wgs/test_aw4_2.csv"
 }
 
 CURATION_PROD_SOURCE_CONFIG = {
   # Optional: to speed up iteration with this script, download and run against a local directory instead.
   :wgs_aw4_prefix => "#{PROD_BROAD_BUCKET}/AW4_wgs_manifest/AoU_DRCB_SEQ_",
-  # The array_old_egt_files should be removed, likely in late 2022. This contains older
-  # AW4 manifests before reprocessing.
-  :microarray_aw4_prefix => "#{PROD_BROAD_BUCKET}/#{OLD_ARRAYS_PATH_INFIX}/AW4_array_manifest/AoU_DRCB_GEN_"
+  # This contains all AW4 manifests.
+  :microarray_aw4_prefix => "#{PROD_BROAD_BUCKET}/AW4_array_manifest/AoU_DRCB_GEN_"
```

Comment:
aw4 files are now available in the new path. The old path shouldn't be used anymore